man/Makefile.am

```diff
@@ -54,7 +54,7 @@ doc_files  = $(addprefix ../docs/man/,$(rst_files))
 # for automake
 # generate man pages only when rst2man is installed
 if HAVE_RST2MAN
-man_MANS = $(man_pages)
+dist_man_MANS = $(man_pages)
```

Comment:
I don't object to this change. However, I would like to point out a minor issue of this change.

configure takes an option, --program-prefix.
You can use the option like:
```
./configure --program-prefix=u-
make
make install 
```
The above command sequences install 'u-ctags'. `u-` is prepended to the command name.

The original man page generator runs after running configure.
The original man page generator reflects the option to the generated man pages.
e.g. in man/ctags-lang-sql.7.rst.in:
```
SYNOPSIS
--------
|       **@CTAGS_NAME_EXECUTABLE@** ... [--extras={guest}] --languages=+SQL ...
```

@CTAGS_NAME_EXECUTABLE@ was replaced.

The replacement done in the original man page generator can be done because it runs after running configure.

`dist_man_MANS` means generated man pages to our tarball.
The man page may be generated after running ./configure by a person who will do "make dist" before running ./configure by another person who does download "universal-ctags.tar.gz" and will do "make".
This is a degradation of the new man page generator with `dist_`.

I think this is acceptable degradation, though.

Reply:
I had to write my intension of this change in the commit log.
I've added the following to it.

>     This allows users who have not installed rst2man to install man pages.

But I am OK to withdraw this change.

***

#1249 was in my mind. 
The prefixed man pages are installed if the users run `configure` with `--prefix` option.

```
$ ./autogen.sh && ./configure --program-prefix=u_ --prefix=/tmp/ctags
...
$ make install
...
make[3]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/tmp/ctags/share/man/man1'
 /usr/bin/install -c -m 644 'ctags.1' '/tmp/ctags/share/man/man1/u_ctags.1'
 /usr/bin/install -c -m 644 'readtags.1' '/tmp/ctags/share/man/man1/u_readtags.1'
...
```

> I think this is acceptable degradation, though.

I also think so, although I overlooked that the replacement of `@CTAGS_NAME_EXECUTABLE@` did not occur as you pointed.
A user has to type `man u_ctags`, and he/she sees the page starting "ctags - Generate tag files for source code".